.github/workflows/ci.yaml

```
@@ -30,7 +30,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [ '1.21', '1.22' ]
+        go: [ '1.21', '1.22', '1.23' ]
```

While you have the hood open, I think it is fine update the matrix for the latest versions. Since the Go min version in go.mod is now Go 1.23 then the toolchain is updating so we are just testing Go 1.23 three times as it is written.

```suggestion
[ '1.23', '1.24' ]
```

---

Done! Waiting for the build